package-lock.json

```diff
@@ -5,6 +5,7 @@
   "requires": true,
```

Comment:
changes resulting from me running `npm install` on node 18. Anyone else get this?

Reply:
Have t tried 18. Have been using 16.